barretenberg/cpp/src/barretenberg/goblin/goblin.hpp

```diff
@@ -205,7 +203,7 @@ class Goblin {
         MergeVerifier merge_verifier;
         bool merge_verified = merge_verifier.verify_proof(proof.merge_proof);
 
-        auto eccvm_verifier = eccvm_composer->create_verifier(*eccvm_builder);
```

Comment:
We didn't really need a constructor of a verifier from a builder so I didn't bother to implement (followed translator model). Ofc it's very easy to add one if/when needed.